system/mp/mpcomm.cpp

```diff
@@ -1889,6 +1914,14 @@ int CMPConnectThread::run()
         }
         if (sock) {
             try {
+                if (soMaxCheck)
+                {
+                    if (numMPAccepts++ >= soMaxConn)
```

Comment:
This seems to be counting the total number rather than the simultaneous number - I don'r really know how the mp stuff works but this thread appears to continue running until the client closes down (do nodes reconnect, or connect more than once) which might lead to spurious warnings I think?